drivers/serial/Kconfig.native_posix

```diff
@@ -41,29 +41,14 @@ endchoice
 
 config UART_NATIVE_WAIT_PTS_READY_ENABLE
 	bool "Support waiting for pseudo terminal client readiness"
-	depends on NATIVE_UART_0_ON_OWN_PTY || UART_NATIVE_POSIX_PORT_1_ENABLE
+	depends on !NATIVE_UART_0_ON_STDINOUT
```

Comment:
This can now not be enabled for uart 1, when uart 0 is on stdin/out.

Reply:
ah, yeah.. the logic didn't work out here.  How do we feel about removing the `depends` on?